package.json

```diff
@@ -25,7 +25,8 @@
     "vue": "^3.3.4",
     "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
-    "vuelidate": "^0.7.4",
+    "@vuelidate/core": "^2.0.3",
+    "@vuelidate/validators": "^2.0.4",
```

Comment:
Since you're updating the main package file, please update the version on line 3.

Reply:
Done.